src/Panel/Panel.less

```diff
@@ -0,0 +1,45 @@
+@import (reference) "../less/index";
+
+.Panel--container {
```

Comment:
nit: should be PascalCase for the child object.

See http://clever.github.io/components/#/less/less-style-guide for explanation